src/SFA.DAS.EmployerIncentives.Data/EarningsResilienceCheck/EarningsResilienceCheckRepository.cs

```diff
@@ -0,0 +1,60 @@
+﻿using SFA.DAS.EmployerIncentives.Abstractions.DTOs.Queries;
+using SFA.DAS.EmployerIncentives.Data.Models;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System;
+using Microsoft.EntityFrameworkCore;
+using SFA.DAS.EmployerIncentives.Domain.IncentiveApplications.Models;
+using SFA.DAS.EmployerIncentives.Data.Map;
+
+namespace SFA.DAS.EmployerIncentives.Data.EarningsResilienceCheck
+{
+    public class EarningsResilienceCheckRepository : IEarningsResilienceCheckRepository
```

Comment:
This repository is working across multiple bounded contexts, it can't.  Shouldn't it just pull all ApplicationApprenticeships where the application has been submitted and the earning calculated flag is false?